include/freerdp/update.h

```diff
@@ -190,6 +190,7 @@ typedef BOOL (*pSaveSessionInfo)(rdpContext* context, UINT32 type, void* data);
 typedef BOOL (*pSetKeyboardImeStatus)(rdpContext* context, UINT16 imeId, UINT32 imeState,
                                       UINT32 imeConvMode);
 typedef BOOL (*pServerStatusInfo)(rdpContext* context, UINT32 status);
+typedef BOOL (*pServerCutText)(rdpContext* context, UINT8* data, UINT32 size);
```

Comment:
why is this necessary in the public API?

Reply:
Hey @akallabeth thanks for reviewing. The reason it's there is the client needs to be able to set it as a call-back for the server to send clipboard updates. I thought it would be a good fit alongside

    instance->update->DesktopResize
    instance->update->BitmapUpdate
    instance->update->BeginPaint
    instance->update->EndPaint

It seemed logical to place it in an equally accessible place.
    instance->update->ServerCutText

That being said, if you think it's not the right place I'm happy to move it as long as i can access rdpClientContext directly from the instance created with freerdp_new() (that returns a freerdp struct). It's not immediately obvious how I can get ahold of rdpClientContext from freerdp.

Reply:
@iiordanov sorry, your answer does not make any sense.
1. what do you mean with `client needs to be able to set it as a call-back for the server to send clipboard updates`?
2. What do you think this callback is for and how is it supposed to work?

Reply:
@iiordanov also, you do not reference that callback anywhere, is this maybe supposed to be part of something else?

Reply:
@akallabeth It's referenced in an app that uses FreeRDP:

https://gitlab.com/iiordanov/remote-desktop-clients-ios/-/blob/master/bVNC/RdpBridge.m#L226

Reply:
@iiordanov ok, did check that. why do you patch this into the core library? it is something that should not be in there as that callback is only used to be called when the clipboard data is available in a desired format?
you are already in the client implementation where this is called and it is very specific for your client, so no, this just looks wrong there.

Reply:
@iiordanov the android implementation you mention also has such stuff encapsulated to the client implementation (e.g. in `androidContext`)

Reply:
Let me try what you suggest @akallabeth .